lib/ui/channel_buffers.dart

```diff
@@ -71,7 +72,7 @@ class _RingBuffer<T> {
     while (_queue.length > lengthLimit) {
       final T item = _queue.removeFirst();
       if (_dropItemCallback != null) {
-        _dropItemCallback(item);
+        _dropItemCallback!(item);
```

Comment:
This can be written as _dropItemCallback?.call(item) instead if desired.  Not sure which is clearer.

Reply:
Neat! Done.